### PR TITLE
Feature/admins controller unit test updated

### PR DESCRIPTION
**Pull request by FahimDev**
@shafayat-bs23 Bhai please review the code and please let me know if everything is okay or not. 

**_This is a  standard Dynamic Unit Test process which can be done with XL/CSV files also. I have used Faker which is also a popular process for Django and Node.js unit test. This can create its own insert operation and run the whole CRUD operation dynamically. So there is no hassle of re-writing or updating values._** 

@antonin686  and @Noman5237  you are requested to follow this process and update this in your branch. Thank you.

Note: @shafayat-bs23  Bhai I have replaced all the bad practices of Response body and made it modular. Please consider this structure until @antonin686  Merge the Central Response structure.

### CODE REVIEW

File: src/admins/admins.controller.spec.ts [hunk 1]
```diff
@@ -17,4 +24,52 @@ describe('AdminsController', () => {
...
+    expect(response.status).toEqual('success');
+    expect(response.data).toBeDefined();
+  });
+
+  it("should find the recently created admin's information", async () => {
+    const response = await controller.findOne(String(adminId));
+    expect(response.status).toEqual('success');
+    expect(response.data).toBeDefined();
+    expect(response.data['name']).toEqual(adminInfo.name);
+  });
```
Comment (shafayat-bs23) [inline src/admins/admins.controller.spec.ts:57]: Make separate methods for different test cases.

**Review by shafayat-bs23 — CHANGES_REQUESTED** [review]
Use mock database and server for testing.

File: src/admins/admins.controller.spec.ts (hunk 1 above)
Comment (FahimDev) [reply src/admins/admins.controller.spec.ts:57]: If I am not wrong each of these are different methods already. And this is the popular process for writing test cases of a single class. We could use 'Describe()' tag. But async was not working properly for that tag.

Comment (shafayat-bs23) [reply src/admins/admins.controller.spec.ts:57]: Definitely all test cases should be in a single class, but you can put different test cases in different methods. Like the following image.

Screenshot 2022-08-22 at 11 16 03 AM

Comment (FahimDev) [reply src/admins/admins.controller.spec.ts:57]: As I have cleared my point, can I Merge the code?

**Pull request merged**